Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -238,7 +245,16 @@ - (void)layoutSubviews {
     CGFloat calendarWidth = CGRectGetWidth(self.calendarView.frame);
     CGFloat weekdayViewHeight = 6 * BPKSpacingMd;
 
-    self.calendarView.frame = CGRectMake(BPKSpacingBase, weekdayViewHeight, width - 2 * BPKSpacingBase, height - weekdayViewHeight);
+    self.calendarView.translatesAutoresizingMaskIntoConstraints = NO;
+    [NSLayoutConstraint activateConstraints:@[
```

Comment:
This is no good for two reasons:

1. We are in `layoutsubviews`, in which we shouldn't be adding constraints
2. We explicitly avoid using Auto Layout in this view because it breaks React Native when we do.